docs/source/en/model_doc/layoutlmv3.mdx

```diff
@@ -39,6 +39,12 @@ alt="drawing" width="600"/>
 
 This model was contributed by [nielsr](https://huggingface.co/nielsr). The TensorFlow version of this model was added by [chriskoo](https://huggingface.co/chriskoo), [tokec](https://huggingface.co/tokec), and [lre](https://huggingface.co/lre). The original code can be found [here](https://github.com/microsoft/unilm/tree/master/layoutlmv3).
 
+## Resources
+
+<PipelineTag pipeline="token-classification"/>
+
+- [`LayoutLMv3ForTokenClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/research_projects/layoutlmv3) and [notebook](https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/LayoutLMv3/Fine_tune_LayoutLMv3_on_FUNSD_(HuggingFace_Trainer).ipynb).
+
```

Comment:
We can also link to the LayoutLMv2 notebooks here, since LayoutLMv3 is almost equivalent to it (and I've got many more notebooks for LayoutLMv2). The only thing to take care of is using `LayoutLMv3Processor` instead of `LayoutLMv2Processor` when preparing data for the model.